internal/handlers/common/update.go

```diff
@@ -236,6 +244,67 @@ func processPopFieldExpression(doc *types.Document, update *types.Document) (boo
 	return changed, nil
 }
 
+// processRenameFieldExpression changes document according to $rename operator.
+// If the document was changed it returns true.
+func processRenameFieldExpression(doc *types.Document, update *types.Document) (bool, error) {
+	renameExpression := update.SortFieldsByKey()
+
+	var changed bool
+
+	for _, key := range renameExpression.Keys() {
+		renameRawValue, err := renameExpression.Get(key)
+		if err != nil {
+			// if $rename field does not exist, don't change anything
+			continue
```

Comment:
Interesting: I see codecov complains about this line, but there is a test case called `NonExistingField` that works with a field that doesn't exist. Do you understand why codecov complains? I wonder if we need a test case like `bson.D{{"$rename", bson.D{{"foo", "boo"}, {"v", "bar"}}}},` where one field doesn't exist and one field exists.

Reply:
@chilagrow you right! Changed it to must.NotFail